app/controllers/application_controller.rb

```diff
@@ -1,3 +1,11 @@
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name, :email, :home_city, :latitude, :longitude])
+
+    devise_parameter_sanitizer.permit(:account_update, keys: [:first_name, :last_name, :email, :home_city, :phone_number, :latitude, :longitude])
```

Comment:
what is this method doing?

Reply:
It's like strong params for other controller but for users with devise, you need to do this to add more fields to sign up and login

Reply:
oh gotcha- are we asking users to input their lat and long? or do those two fields just have to be included here to be generated once home city is inputted?

Reply:
Lat and long will be auto generated by geocoder or google maps api but need to be there to put in the database when users input home city